src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -283,6 +283,10 @@
 helps['storage account management-policy update'] = """
 type: command
 short-summary: Update the data policy rules associated with the specified storage account.
+examples:
+    - name: Update the data policy rules associated with the specified storage account.
+      text: |
+        az storage account management-policy update --account-name myaccount --resource-group myresourcegroup --add property.listProperty
```

Comment:
Could you see the change in response with the new added property?

Reply:
could you also add --set examples because customer wants to know how to set the policy as you can see in the issue?

Reply:
https://github.com/Azure/azure-cli/blob/bcb291ad55218dd8d7d582245d4fe97beb916309/src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py#L739-L740